### DISCUSSION TITLE
STLINK-V3PWR

### DISCUSSION THREAD

**Discussion by cederom** [stlink-org/stlink#1432] [Q&A (resolved)] [answered]
Hello world :-)

Are there any plans to support STLINK-V3PWR [1]? This is STLINK but with precision Power Profiling features and some additional peripherals :-)

[1] https://www.st.com/en/development-tools/stlink-v3pwr.html

**Comment by Nightwalker-87** [comment]
@cederom: There are no specific plans yet.
However, I found that at least some initial preparations for it have already made it into the codebase (see #1389).

**Reply by Nightwalker-87** [reply]
You may give it a try though using the latest `testing` branch and report on your findings.

**Reply by Nightwalker-87** [reply, marked as answer]
Please open a new issue for a feature request if this is still an issue by now.
